eng/pipelines/repo-mirror.yml

```diff
@@ -0,0 +1,121 @@
+trigger: none
+pr: none
+
+# Sync every 2 hours
+schedules:
+  - cron: "0 */2 * * *"
```

Comment:
Lets avoid running this every 2 hours. I think doing it daily should generally be enough and if there are any times we need it done immediately we can run the job manually.